[Bug]: Linting fails on main: time-naming violation in search_traces.go

### What happened?

Running `make lint` on the main branch fails, preventing local development and contributions. The linting process exits with an error about incorrect variable naming in search_traces.go, which blocks anyone trying to build or test the project locally.

### Steps to reproduce

1. Clone the repository
2. Checkout the latest main branch (commit 3361fcc4 or later)
3. Run `make lint`
4. Observe the linting failure with error about durationMin variable naming

### Expected behavior

`make lint` should pass without errors on the main branch.

### Relevant log output

```shell
INFO [runner] linters took 866.950583ms with stages: goanalysis_metalinter: 833.429466ms 
cmd/jaeger/internal/extension/jaegermcp/internal/handlers/search_traces.go:124:6: time-naming: var durationMin is of type time.Duration; don't use unit-specific suffix "Min" (revive)
        var durationMin, durationMax time.Duration
            ^
```

### Screenshot

<img width="1530" height="359" alt="Image" src="https://github.com/user-attachments/assets/9fa7ce7c-f234-4f1b-bbce-a2d8782dd3e4" />

### Additional context

_No response_

### Jaeger backend version

_No response_

### SDK

_No response_

### Pipeline

_No response_

### Stogage backend

_No response_

### Operating system

_No response_

### Deployment model

_No response_

### Deployment configs

```shell

```

--- a/cmd/jaeger/internal/extension/jaegermcp/internal/handlers/search_traces.go
+++ b/cmd/jaeger/internal/extension/jaegermcp/internal/handlers/search_traces.go
@@ -101,19 +101,19 @@ func (*searchTracesHandler) buildQuery(input types.SearchTracesInput) (querysvc.
 	}
 
 	// Parse and validate input
-	startTimeMin, err := parseTimeParam(startTimeMinInput)
+	minStartTime, err := parseTimeParam(startTimeMinInput)
 	if err != nil {
 		return querysvc.TraceQueryParams{}, fmt.Errorf("invalid start_time_min: %w", err)
 	}
 
-	var startTimeMax time.Time
+	var maxStartTime time.Time
 	if input.StartTimeMax != "" {
-		startTimeMax, err = parseTimeParam(input.StartTimeMax)
+		maxStartTime, err = parseTimeParam(input.StartTimeMax)
 		if err != nil {
 			return querysvc.TraceQueryParams{}, fmt.Errorf("invalid start_time_max: %w", err)
 		}
 	} else {
-		startTimeMax = time.Now()
+		maxStartTime = time.Now()
 	}
 
 	if input.ServiceName == "" {
@@ -165,8 +165,8 @@ func (*searchTracesHandler) buildQuery(input types.SearchTracesInput) (querysvc.
 			ServiceName:   input.ServiceName,
 			OperationName: input.OperationName,
 			Attributes:    attributes,
-			StartTimeMin:  startTimeMin,
-			StartTimeMax:  startTimeMax,
+			StartTimeMin:  minStartTime,
+			StartTimeMax:  maxStartTime,
 			DurationMin:   durationMin,
 			DurationMax:   durationMax,
 			SearchDepth:   searchDepth,

